### PR TITLE
Password reset page

### LINKED ISSUE #2813: Fix broken reset password page inside app
### Expected Behavior
As a student, I expect to be able to fill out a form to reset my password.

### Actual Behavior
Password reset form page styling is broken and form elements are partially hidden behind header.

### Steps to reproduce
[Trello Card](https://trello.com/c/coafl3Go/72-fix-the-unfinished-broken-style-for-resetting-your-password)
Branch: `password-reset-page`
URL: http://localhost:5000/passwords/reset

### PR DESCRIPTION
**Pull request by mariehooper**
### Status
**READY**

### Description
This PR restyles the password rest page for users who request a new password as well as removes a legacy view, route and controller method for requesting a new password inside app.

### Migrations
NO

### Impacted Areas in Application
List general components of the application that this PR will affect:

* Password reset page (tokenized link from email)

Closes #2813

### CODE REVIEW

File: app/models/breadcrumb_trail.rb [hunk 1]
```diff
@@ -408,6 +408,10 @@ def grades_importers_import_results
     breadcrumb('Dashboard', dashboard_path)
     breadcrumb('#{ term_for :assignments }', assignments_path)
   end
+  
+  def passwords_new
+    breadcrumb('Dashboard', dashboard_path)
```
Comment (chcholman) [inline app/models/breadcrumb_trail.rb]: Seems like we should be able to remove this method if we've deleted the rest of the `new` method?

Comment (mariehooper) [inline app/models/breadcrumb_trail.rb]: yep! will do.

**Comment by chcholman** [comment]
@mariehooper Finally found the last page - account activation also has a password set form. Pulled in your styles to the form, but we can refactor to have both pages reference the same view partial next!

**Comment by mariehooper** [comment]
@chcholman - got it! Thank you, Cait!

**Pull request merged**